pkg/controller/controller_binding_test.go

```diff
@@ -1473,3 +1473,109 @@ func TestReconcileUnbindingWithServiceBrokerHTTPError(t *testing.T) {
 		t.Fatalf("Received unexpected event: %v, expecting: %v", a, e)
 	}
 }
```

Comment:
Should have assertions that enforce that the operation is for the unbind success case, and that it is set int eh error case when you are not done doing work.